app/keepers/keepers.go

```diff
@@ -212,7 +216,13 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 		appKeepers.ScopedIBCKeeper,
 	)
 
-	appKeepers.WireICS20PreWasmKeeper(appCodec, bApp)
+	// Configure the hooks keeper
+	hooksKeeper := ibchookskeeper.NewKeeper(
+		appKeepers.keys[ibchookstypes.StoreKey],
+	)
+	appKeepers.IBCHooksKeeper = &hooksKeeper
+
+	appKeepers.WireICS20PreWasmKeeper(appCodec, bApp, appKeepers.IBCHooksKeeper)
```

Comment:
What's the reason for adding these manually instead of cherry picking https://github.com/osmosis-labs/osmosis/pull/3589/ ?

Reply:
It might have just been my confusion dealing with multiple cherry picks